src/docs/resources/faq.md

```diff
@@ -84,8 +84,7 @@ apps are in the works, or "inside the firewall".
 
 Teams outside of Google are also using Flutter.
 For example, the official app for the musical Hamilton
-([Android](https://play.google.com/store/apps/details?id=com.hamilton.app),
-[iOS](https://itunes.apple.com/us/app/hamilton-the-official-app/id1255231054?mt=8))
+([Android][Hamilton for Android], [iOS][Hamilton for iOS])
```

Comment:
I know that the change to this question was outside the scope of the issue that you've fixed, but ... IMHO this question should merely be a segway to the Showcase. The entire answer could be:

> Developers both inside and outside Google are using Flutter to build beautiful native business-critical apps for iOS and Android. Discover some of these apps by visiting the Flutter [Showcase](/showcase).

Or, if you prefer, we can borrow some of the introductory Showcase text:

> Flutter is used by Google and developers and organizations around the world to deliver beautiful native apps on iOS and Android. Learn about some of these apps from the Flutter [Showcase](/showcase).

(What originally got me thinking about this change is the following: We don't provide any download links in the first paragraph, so why do so here?)

Reply:
I really like the idea of linking to the Showcase here.